build-tools/packages/build-cli/src/commands/release.ts

```diff
@@ -0,0 +1,834 @@
+
+                if (this.bumpType === undefined) {
+                    const choices = [
+                        { value: "major", name: `major (${currentVersion} => ${bumpedMajor})` },
+                        { value: "minor", name: `minor (${currentVersion} => ${bumpedMinor})` },
+                        { value: "patch", name: `patch  (${currentVersion} => ${bumpedPatch})` },
+                    ];
+                    this.bumpType = getDefaultBumpTypeForBranch(currentBranch) ?? this.bumpType;
+                    const askBumpType: inquirer.ListQuestion = {
+                        type: "list",
+                        name: "bumpType",
+                        choices,
+                        default: this.bumpType,
+                        message: `The current branch is '${currentBranch}'. The default bump type for that branch is '${this.bumpType}', but you can change it now if needed.`,
+                    };
+                    questions.push(askBumpType);
+                    const answers = await inquirer.prompt(questions);
+                    this.bumpType = answers.bumpType;
+                }
+
+                this.logHr();
+                this.log(`RELEASE PLAN\n`);
+                this.log(`Release group: ${this.releaseGroup}`);
+                this.log(`Version to be released: ${chalk.greenBright(currentVersion)}`);
+                this.log(`Bump type: ${this.bumpType}`);
+                this.log(
+                    `Version after bump: ${bumpVersionScheme(currentVersion, this.bumpType!)}`,
+                );
+
+                this.machine.action("success");
+                break;
+            }
+
+            case "DoChecks": {
+                this.machine.action("success");
+                break;
+            }
+
+            case "CheckShouldRunOptionalChecks": {
+                if (this.shouldSkipChecks) {
+                    this.machine.action("failure");
+                }
+
+                this.machine.action("success");
+                break;
+            }
+
+            case "CheckValidReleaseGroup": {
+                if (isReleaseGroup(this.releaseGroup)) {
+                    this.machine.action("success");
+                    // eslint-disable-next-line no-negated-condition
+                } else if (context.fullPackageMap.get(this.releaseGroup!) !== undefined) {
+                    this.machine.action("success");
+                } else {
+                    this.machine.action("failure");
+                }
+
+                break;
+            }
+
+            case "CheckPolicy": {
+                if (this.shouldCheckPolicy) {
+                    if (context.originalBranchName !== "main") {
+                        this.warn(
+                            "WARNING: Policy check fixes are not expected outside of main branch!  Make sure you know what you are doing.",
+                        );
+                    }
+
+                    // TODO: Call new check policy command
```

Comment:
I'm assuming this is still not actionable?

Reply:
Change is made, coming in future PR.